modules/apps/dispatch/dispatch-web/src/main/resources/META-INF/resources/trigger/details.jsp

```diff
@@ -35,7 +35,7 @@ if (dispatchTrigger != null) {
 
 <portlet:actionURL name="/dispatch/edit_dispatch_trigger" var="editDispatchTriggerActionURL" />
 
-<div class="closed container-view" id="<portlet:namespace />editDispatchTriggerId">
+<div class="closed container-fluid container-fluid-max-xl container-form-lg" id="<portlet:namespace />editDispatchTriggerId">
```

Comment:
Probably we could use new layout taglibs here!

Reply:
@marcoscv-work I updated.